src/arithmetic/ibex_IntervalVector.cpp

```diff
@@ -46,6 +46,11 @@ IntervalVector::IntervalVector(int n1, double bounds[][2]) : n(n1), vec(new Inte
 			vec[i]=Interval(bounds[i][0],bounds[i][1]);
 }
 
+IntervalVector::IntervalVector(std::initializer_list<Interval> list) : n(list.size()), vec(new Interval[n]) {
+	assert(n >= 1);
+	std::copy(list.begin(), list.end(), vec);
+}
+
```

Comment:
Est-ce normal que le paramètre du constructeur soit passé par copie? (j'ai la même question en fait pour tous les constructeurs que tu as ajoutés...)

Reply:
Je me suis aussi posé la question, voici une des réponses que j'ai trouvée : https://stackoverflow.com/questions/17803475/why-is-stdinitializer-list-often-passed-by-value
En gros, un initializer_list est comme un intérateur ou un pointeur, pour lesquelles une copie est en général plus rapide qu'un passage par référence, et le meilleur exemple c'est que la librairie standard passe tout ses initializer_list par valeur. 
En plus c'est un idiome tellement commun que je pense que tous les compilateurs optimisent très bien ce genre de construction.

Reply:
Effectivement et merci pour le lien. Et s'ils font comme ça dans la std, c'est signe que tu as clairement raison ;)